jps-plugin/src/org/jetbrains/jps/incremental/scala/local/CompilerFactoryImpl.scala

```diff
@@ -97,9 +96,9 @@ object CompilerFactoryImpl {
                                client: Option[Client]): File = {
 
     val scalaVersion = scalaInstance.actualVersion
-    val sourceJar =
-      if (isBefore_2_11(scalaVersion)) sourceJars._2_10
-      else sourceJars._2_11
+    val sourceJar = if(scalaVersion.contains("hydra")) scalaInstance.otherJars().find(_.getName.contains("hydra-bridge")).get
```

Comment:
Don't call `.get`, it will crash the builder with `NoSuchElementException`. If `hydra-bridge` isn't there it's indeed a bug, but we need to be robust and fallback to the regular code path.